### DISCUSSION TITLE
Why doesn't it work when I modify the theme's html?

### DISCUSSION THREAD

**Discussion by yigeqiji** [CaiJimmy/hugo-theme-stack#663] [Q&A] [answered]
### What happened?

When I modify `hugo-theme-stack/layouts/partials/article-list/compact.html`, nothing seems to work, am I missing something important?
What should I do to modify the style of the home page list correctly?

### Hugo version

ex 0.100.0

### Theme version

ex .3.

### What browsers are you seeing the problem on?

Microsoft Edge

### More information about the browser

windows11

### Link to the page with bug

https://demo.stack.jimmycai.com

**Comment by CaiJimmy** [comment, marked as answer]
> Link to the source repository
> No response

Provide the website's source code, otherwise, nobody knows why it fails.

**Reply by yigeqiji** [reply]
> > Link to the source repository
> > No response
> 
> Provide the website's source code, otherwise, nobody knows why it fails.

Sorry, let me give you an example, how can I change the click area of the home page list module, now I can only go to the details by clicking on the title or the image,a large blank area in the lower right corner is not clickable.
![image](https://user-images.githubusercontent.com/59365443/187139372-020f2dff-f0f1-4b4e-b1c4-171bcdbd7416.png)
What should I do
